src/kbmod/search/raw_image.cpp

```diff
@@ -331,6 +331,28 @@ ImageMoments RawImage::find_central_moments() const {
     return res;
 }
 
+bool RawImage::center_is_local_max(double flux_thresh, bool local_max) const {
+    const int num_pixels = width * height;
+    int c_x = width / 2;
+    int c_y = height / 2;
+    int c_ind = c_y * width + c_x;
+
+    auto pixels = image.reshaped();
+    double center_val = pixels[c_ind];
+
+    // Find the sum of the zero-shifted (non-NO_DATA) pixels.
+    double sum = 0.0;
+    for (int p = 0; p < num_pixels; ++p) {
+        float pix_val = pixels[p];
```

Comment:
Optional change. You could either remove this line and use `pixels[p]` in the line below. Or keep the line and use `pix_val` on line 350.